ark/segmentation/marker_quantification.py

```diff
@@ -27,11 +29,20 @@ def compute_marker_counts(input_images, segmentation_labels, nuclear_counts=Fals
             morphology features for regionprops to extract for each cell
         split_large_nuclei (bool):
             controls whether nuclei which have portions outside of the cell will get relabeled
+        extraction (str):
+            extraction function used to compute marker counts. default is 'total_intensity'
+        **kwargs:
+            arbitrary keyword arguments
     Returns:
         xarray.DataArray:
             xarray containing segmented data of cells x markers
     """
 
+    misc_utils.verify_in_list(
+        extraction=[extraction],
```

Comment:
You don't need to put `extraction` in a list for `verify_in_list` to work.

Reply:
I'll try it without that, but my intuition told me that python will regard that string as a list of characters.

Reply:
Right, I forgot it was a string.